lib/SILOptimizer/Transforms/ArrayElementValuePropagation.cpp

```diff
@@ -53,23 +67,32 @@ class ArrayAllocation {
 
   // The calls to Array get_element that use this array allocation.
   llvm::SmallSetVector<ApplyInst *, 16> GetElementCalls;
+  SmallVector<ApplyInst *, 4> AppendContentsOfCalls;
```

Comment:
Please be consistent about using llvm:: prefix. In general in the optimizer we use it for SmallVector and SmallVectorImpl (in fact you even did that below).

I would look at what the file is doing and match that.